.github/ISSUE_TEMPLATE/bug-performance-report.md

```diff
@@ -5,24 +5,18 @@ about: Use this template for reporting a bug or a performance issue.
 ---
 
 **System information**
-- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
 - OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
 - TensorFlow installed from (source or binary):
-- TensorFlow version (use command below):
-- TensorFlow Addons installed from (source, PyPi):
+- TensorFlow Addons installed from (source or binary):
```

Comment:
move one line up? We might want to put `TensorFlow installed from` and `TensorFlow version` together.

Reply:
How do you feel about the change?